_data/CSElonewolf/CSElonewolf.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Devkant Gorai 
+institution: Government College of Engineering and Textile Technology,Berhampore 
```

Comment:
The name of the institution must **not** be longer than 58 characters, yours is **67** characters.